### PR TITLE
Task 8

### PR DESCRIPTION
**Pull request by Viccart**

### CODE REVIEW

File: src/components/AddComment.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+export default function AddComment({ articleId, isLoading, setComments }) {
+  const [commentBody, setCommentBody] = useState("");
+  const [isSubmitted, setIsSubmitted] = useState(false);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log("form submitted");
```
Comment (B7Bprog) [inline src/components/AddComment.jsx]: Please remove all logs from this final code.

**Review by B7Bprog — CHANGES_REQUESTED** [review]
Hey Hannah, I really like the design of your site.

Here are a few things to fix/refactor:

- The logged-in user should be indicated on each page.
- User should be able to use the form and post again after a comment had been posted. Without refreshing the page.
- An error message should be displayed when the post request is unsuccessful.
- The form should be disabled for the time of posting.

Please add these functionalities, then send us another PR with it.

File: src/api.js [hunk 2]
```diff
@@ -40,4 +40,19 @@ export const patchArticle = (id) => {
   });
 };
 
+export const postComment = (newComment) => {
+  console.log(newComment);
```
Comment (callumu21) [inline src/api.js:44]: This console.log should be removed

File: src/components/AddComment.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+export default function AddComment({ articleId, setComments, currentUser }) {
+  const [commentBody, setCommentBody] = useState("");
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const [err, setErr] = useState("");
+
+  const handleSubmit = (e) => {
+    setIsSubmitted(true);
```
Comment (callumu21) [inline src/components/AddComment.jsx:10]: I would rename this state to isSubmitting to illustrate you're tracking the progress of a submit, not whether a user has submitted successfully.

**Review by callumu21 — APPROVED** [review]
Really nice job implementing Béla's changes. I've left a couple of comments for minor changes, but this is good to merge now 👍 

As a minor bit of styling for later, you may want to resize the textarea for a comment body depending on the screen size, maybe keeping it in line with the comment cards for a consistent look on the page.

**Pull request merged**